ur5e_moveit_config/launch/run_move_group.launch.py

```diff
@@ -0,0 +1,145 @@
+import os
+import yaml
+from launch import LaunchDescription
+from launch_ros.actions import Node
+from ament_index_python.packages import get_package_share_directory
+import xacro
+
+def load_file(package_name, file_path):
+    package_path = get_package_share_directory(package_name)
+    absolute_file_path = os.path.join(package_path, file_path)
+
+    try:
+        with open(absolute_file_path, 'r') as file:
+            return file.read()
+    except EnvironmentError: # parent of IOError, OSError *and* WindowsError where available
+        return None
+
+def load_yaml(package_name, file_path):
+    package_path = get_package_share_directory(package_name)
+    absolute_file_path = os.path.join(package_path, file_path)
+
+    try:
+        with open(absolute_file_path, 'r') as file:
+            return yaml.safe_load(file)
+    except EnvironmentError: # parent of IOError, OSError *and* WindowsError where available
+        return None
+
+
+def generate_launch_description():
+
+    # planning_context
+    # set ur robot
+    robot_name = 'ur5e'
+
+    # <robot_name> parameters files
+    joint_limits_params = os.path.join(get_package_share_directory('ur_description'), 'config/' +
+                                       robot_name, 'joint_limits.yaml')
+    kinematics_params = os.path.join(get_package_share_directory('ur_description'), 'config/' +
+                                     robot_name, 'default_kinematics.yaml')
+    physical_params = os.path.join(get_package_share_directory('ur_description'), 'config/' +
+                                   robot_name, 'physical_parameters.yaml')
+    visual_params = os.path.join(get_package_share_directory('ur_description'), 'config/' +
+                                 robot_name, 'visual_parameters.yaml')
+
+    # common parameters
+    # If True, enable the safety limits controller
+    safety_limits = False
```

Comment:
shouldn't this be in some YAML file? If this file is generated in the future by MSA, then it is very hacky to add variables like this.

Reply:
I didn't want to implement yaml parsing here because this is parameter that is passed to the urdf and xacro files for additional parsing.